blocks/serializer.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import * as query from 'hpq';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockSettings } from './registration';
+
+/**
+ * Takes a block list and returns the serialized post content
+ *
+ * @param  {Array}  blocks Block list
+ * @return {String}        The post content
+ */
+export default function serialize( blocks ) {
+	return blocks.reduce( ( memo, block ) => {
+		const blockType = block.blockType;
+		const blockSettings = getBlockSettings( blockType );
+
+		// static content, to be rendered inside the block comment
+		const rawContent = blockSettings.serialize( block.attributes );
+
+		// To compute the blocks attributes we need serialize as comment attributes
+		// We take all the block attributes and exclude the block attributes computed
+		// using the `attributes` from the Block Settings.
+		let contentAttributes = {};
+		if ( 'function' === typeof blockSettings.attributes ) {
```

Comment:
Should we abstract this since it's now both here and in `parser`'s `getBlockAttributes` ? (Or rather, should we just move `getBlockAttributes` somewhere more generic?).